packages/manager/modules/cloud-connect/src/index.js

```diff
@@ -0,0 +1,52 @@
+import angular from 'angular';
+
+import '@ovh-ux/manager-core';
+import '@uirouter/angularjs';
+import 'angular-translate';
+import 'oclazyload';
+import 'ovh-api-services';
+import '@ovh-ux/ng-at-internet';
+import moment from 'moment';
+import ngOvhCloudUniverseComponents from '@ovh-ux/ng-ovh-cloud-universe-components';
+
+import './style.scss';
+
+const moduleName = 'ovhCloudConnectLazyLoading';
+
+angular
+  .module(moduleName, [
+    'ovhManagerCore',
+    'pascalprecht.translate',
+    'ui.router',
+    'oc.lazyLoad',
+    'ovh-api-services',
+    ngOvhCloudUniverseComponents,
+  ])
+  .config(
+    /* @ngInject */ ($stateProvider) => {
+      $stateProvider.state('cloud-connect.**', {
+        url: '/cloud-connect/:ovhCloudConnectId',
```

Comment:
```suggestion
        url: '/cloud-connect',
```
I think root state should be reserved for listing page even if there are none yet